tests/integration/it_employees_test.py

```diff
@@ -63,9 +69,10 @@ def create_employee():
                     password="bla",
                     pin_code=1234,
                     comment="No comments",
-    )
+                    )
 
 
+@pytest.mark.skip(reason="Authentication injection not implemented")
```

Comment:
@paulodamaso can you also implement a test with authentication, not only Forbidden cases? So you know it actually works and there will also be an example for later. 

If you don't have anymore time, then maybe add more details to the puzzle above? How should the tests be modified to support authentication?

Reply:
@amihaiemil I've already left the puzzle for this task, now it's correct :)